.github/workflows/dev.yaml

```diff
@@ -1,79 +1,78 @@
 name: Build, Tag, Publish and Deploy to GKE
 
 on:
-  push:
-    branches:
-      - dev
+    push:
+        branches:
+            - dev
 
 # Environment variables available to all jobs and steps in this workflow
 env:
-  GITHUB_SHA: ${{ github.sha }}
-  DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-  DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
-  DOCKER_ORGANIZATION: ${{ secrets.DOCKER_ORGANIZATION }} 
-  PROJECT_ID: ${{ secrets.GKE_PROJECT }}
-  GKE_CLUSTER: wisp-prod
-  GKE_ZONE: northamerica-northeast1-c
```

Comment:
Change GKE_ZONE to be a secret while we're at it

Reply:
Good catch, updated!